pyproject.toml

```diff
@@ -63,7 +63,7 @@ phonenumbers = "^8.13.38"
 
 [tool.black]
 line-length = 120
-target-version = ['py38']
+target-version = ['py311']
```

Comment:
needed to allow for the `match` syntax to be parsed

Reply:
Oh we should have changed this anyway. Would you mind have a quick look around the codebase for any other references to old Python versions? I can see there's one to 3.9 in the README, for example.

Reply:
had a look through and I think that's the only remaining reference, so have fixed both